### DISCUSSION TITLE
Works on 3468A with few modifications

### DISCUSSION THREAD

**Discussion by NietoSkunk** [xyphro/HP3457-OLED-display#3] [General]
I just finished building this display on a HP 3468A multimeter, and after a few modifications, it worked perfectly!

![IMG_7779](https://github.com/xyphro/HP3457-OLED-display/assets/50814274/32f857d3-c47b-448c-a08b-8fcd63ab227c)

Hardware wise, there were no modifications necessary on the PCB itself. I did have to replace the ISO7240MDW with a ISO7740DWR for component supply reasons, but it was pin compatible so there were no issues. I had to find some different pads to pull a 12V source from, and I selected one side of CR711 and CR712 for +15V and GND respectively. Additionally, I had to short R564 since the 1K resistor was causing too much voltage drop on the 5V reference. Finally, I added a slight ND gel filter behind the clear window to hide the OLED screen edges.

![IMG_7778](https://github.com/xyphro/HP3457-OLED-display/assets/50814274/f1d2714a-d79e-41eb-be32-ddb0bf80ad02)
![IMG_7777](https://github.com/xyphro/HP3457-OLED-display/assets/50814274/2f9bcef0-ff09-42ea-ade2-b0e9f0fc0ea2)
![IMG_7776](https://github.com/xyphro/HP3457-OLED-display/assets/50814274/df717a53-d255-4909-ae7b-17a257b16084)
![IMG_7775](https://github.com/xyphro/HP3457-OLED-display/assets/50814274/09f4d649-a3cb-4d23-86e3-92dec6be0623)

Software wise, I only had to change the `ANNUCIATORS_PERSONALITY` to `HP3478_ICONS` and bump the painting offset from 53 to 63 to be centered within the display. 

Thank you so much for this project! I'm glad to be able to see my multimeter screen at an angle now.

**Comment by xyphro** [comment]
Really great Build, I love it!

Thanks a lot for putting it here, now we know it can be used for HP3468A
too.


NietoSkunk ***@***.***> schrieb am So., 17. Sept. 2023, 21:39:
